pkg/sql/alter_schema.go

```diff
@@ -26,7 +28,7 @@ import (
 
 type alterSchemaNode struct {
 	n    *tree.AlterSchema
-	db   *sqlbase.MutableDatabaseDescriptor
+	db   *dbdesc.MutableDatabaseDescriptor
```

Comment:
Is the goal to make this `dbdesc.MutableDescriptor`?

Reply:
have not decided, do you want it to be?

Reply:
maybe once we have packages for each kind of descriptor